conanfile_base.py

```diff
@@ -0,0 +1,104 @@
+import os
+import glob
+import platform
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class ConanfileBase(ConanFile):
+    _base_name = "flex"
+    name = "flex"
+    version = "2.6.4"
+    url = "https://github.com/bincrafters/conan-flex"
+    homepage = "https://github.com/westes/flex"
+    description = "Flex, the fast lexical analyzer generator"
+    topics = ("conan", "flex", "lex", "lexer", "lexical analyzer generator")
+    license = "BSD-2-Clause"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    exports = ["LICENSE.md"]
```

Comment:
```suggestion
    exports = ["LICENSE.md", os.path.basename(__file__)]
```

Reply:
I see the point now, you are using this recipe as base and also as library package ... :+1: